atorch/examples/llama2/README.md

```diff
@@ -423,4 +423,98 @@ sh ds_3d_llama2_entry.sh
 | DS 3D | /        |           8 |  2 |  2 |  2 |             8 |              64 |              1024 | 155.73 | 49.9 | 31132.3               |
 | DS 3D | /        |          16 |  2 |  2 |  4 |             8 |              64 |              2048 | 154.84 | 49.6 | 26309.4               |
 | DS 3D | /        |          32 |  2 |  2 |  8 |             8 |              64 |              4096 | 154.44 | 49.5 | 23901.8               |
-| DS 3D | /        |          64 |  2 |  2 | 16 |             8 |              64 |              8192 | 153.93 | 49.3 | 22695.7               |
\ No newline at end of file
+| DS 3D | /        |          64 |  2 |  2 | 16 |             8 |              64 |              8192 | 153.93 | 49.3 | 22695.7               |
+
+## Automatic Training Optimization 
+### Introductin
+If the users are not sure what the strategy to achieve the largest throughput, auto_accelerate is able to automatically searching the best strategy given models and hardware conditions. This function is based on Bayesian Optimization (BO)implemented by [HEBO](https://github.com/huawei-noah/HEBO) aiming to find the strategy with largest training throughput efficiently. BO is a machine learning technique used for optimizing black-box functions that are expensive to evaluate. Specifically, BO learn the mapping from strategies to throughput using the data from dryrun, which run few training steps. Moreover, BO recommends the potential high-throughput strategy to achieve the throughput from dryrun, and update the mapping iteratively. This iterative process continues until the desired optimization criteria are met or a predefined budget is exhausted.
+
+### Usage
+In this section,an example is presented to show how to automatically search the best strategy. Specifically, we utilize 8 A100 to train the LLAMA2 7B model on wikitext-103-raw-v1. The shell script to run the experiment is given as follows
+```bash
```

Comment:
This is the script in bayes_opt_sg_llama2_entry.sh ? Then just say that : the example source code is [bayes_opt_sg_llama2.py](bayes_opt_sg_llama2.py) and you can use [bayes_opt_sg_llama2_entry.sh](bayes_opt_sg_llama2_entry.sh) to run the example.
Then explains the env variables and auto_acceleate API parameters.